src/index.js

```diff
@@ -0,0 +1,18 @@
+// import _ from 'lodash';
```

Comment:
Try not to commit commented code. Unless you need it in the future and then comment with `TODO: uncomment code afther ...`

Reply:
Yep the first eleven lines of code I should have deleted. But 16 and 17 line shows that it works fine with two options of writing code.

Reply:
Do not afraid to rewrite your code or delete some code, you can always see the PR history.